roles/docker_tag/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+#
+#  Tags an image
+#
+
+- name: Fail if dt_image or dt_tag is not defined
+  fail:
+    msg: "dt_image or dt_tag is not defined"
+  when: dt_image is undefined or
+        dt_tag is undefined
+
+- debug:
+    msg: "dt_img: {{ dt_image }}"
+
+- debug:
```

Comment:
Did you want to leave these `debug:` statements here?

Reply:
Nope, removed!